backend/mora/settings.py

```diff
@@ -52,13 +52,7 @@
 
 PROD_MODE = False
 
-USER_SETTINGS = {
-    'orgunit': {
-        'show_location': True,
-        'show_user_key': False,
-        'show_roles': True
-    }
-}
+USER_SETTINGS_DB_FILE = '/tmp/usersettings.db'
```

Comment:
I don't think `/tmp` is a suitable default; everyone can write there, and it may get cleaned on boot. Use something relative to `$PWD` instead?